native/cocos/platform/android/java/src/com/cocos/lib/CocosVideoHelper.java

```diff
@@ -156,6 +160,27 @@ public void handleMessage(Message msg) {
                 helper._setVolume(msg.arg1, volume);
                 break;
             }
+            case VideoTaskSetPlaybackRate: {
+                float rate = (float) msg.arg2 / 10;
```

Comment:
Why divide by 10 here, then multiply 10 in `helper._setPlaybackRate()`?

Reply:
![image](https://github.com/cocos/cocos-engine/assets/18634782/ede526a0-36de-4310-909b-743b790865ef)
Because`arg` type is `int`.

Reply:
got it